scripts/launch/mainnet/LidoAddressesMainnet.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.26;
+
+contract LidoAddressesMainnet {
+    address public constant ACL = 0x9895F0F17cc1d1891b6f18ee0b483B6f221b37Bb;
+    address public constant LIDO = 0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84;
+    address public constant KERNEL = 0xb8FFC3Cd6e7Cf5a098A1c92F48009765B24088Dc;
+    address public constant VOTING = 0x2e59A20f205bB85a89C53f1936454680651E618e;
+    address public constant TOKEN_MANAGER = 0xf73a1260d222f447210581DDf212D915c09a3249;
+    address public constant FINANCE = 0xB9E5CBB9CA5b0d659238807E84D0176930753d86;
+    address public constant AGENT = 0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c;
+    address public constant EVM_SCRIPT_REGISTRY = 0x853cc0D5917f49B57B8e9F89e491F5E18919093A;
```

Comment:
please add proof for EVM_SCRIPT_REGISTRY

Reply:
https://etherscan.io/address/0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c#readProxyContract#F24